bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/library/items/RollershutterItem.java

```diff
@@ -37,6 +37,11 @@
 
     static {
         acceptedDataTypes.add(PercentType.class);
+		/*

```

Comment:
Honestly, I'm not really sure whether this is the right place for such an explanation. It's not even shown in the JavaDoc for RollershutterItem. Somebody would need to look inside the sources and scroll to the right position in order to find it. 

IMHO the [items documentation](https://github.com/eclipse/smarthome/blob/master/docs/documentation/concepts/items.md) would be the much better place for such explanations.